tests/validation/tests/v3_api/test_rke_cluster_provisioning.py

```diff
@@ -221,14 +293,15 @@
 
 def test_cis_complaint():
```

Comment:
Add code here to support cluster creation for both `rke-cis-1.4` and `rke-cis-1.5`.

Reply:
`rke_config_cis = get_cis_rke_config()`
I still need to include a couple of manual steps unique to 1.5